### PR TITLE
Made file submission box VPAT compliant

### LINKED ISSUE #3155: VPAT — Keyboard user cannot navigate to file submission box
Tab key cannot gain focus to the file submission box that is rendered for gradeable submissions.  This prevents keyboard-only users from submitting any files for gradeables.

Submission box rendering code:
https://github.com/Submitty/Submitty/blob/c99b355092dd43cfae0babee2fce28e9c54d744b/site/app/templates/submission/homework/SubmitBox.twig#L129-L142

### PR DESCRIPTION
**Pull request by Kevin-MM**
Closes #3155

### What is the current behavior?
Previously when using the tab key to navigate through items on the page file submission boxes were passed over, making it impossible for keyboard only users to submit files.

### What is the new behavior?
File submission boxes can now receive focus when using the tab key.  When a file submission box has focus and the enter key is pressed the file selector appears.

### Other information?
Mostly tested with Development -> Choice Of Language and also with other development grades that utilize a single file submission box.

### CODE REVIEW

File: site/app/templates/submission/homework/SubmitBox.twig [hunk 1]
```diff
@@ -225,13 +225,13 @@
                             {% endif %}
                         </div>
                         {% set num_inputs = num_inputs + 1 %}
-                    {% endfor %}
+                    {% endfor %}_
```
Comment (emaicus) [inline site/app/templates/submission/homework/SubmitBox.twig]: Is this an unintentional underscore?

Comment (Kevin-MM) [reply site/app/templates/submission/homework/SubmitBox.twig]: Yes, I've removed it.

**Comment by Kevin-MM** [comment]
Ok, so I've made many changes to this PR.

In addition to the original details,

1) When a file submission box has focus a blue outline appears around the box indicating so
2) After a file has been uploaded the trash can that appears next to the filename is now accessible by pressing tab.  The trash can will also receive a blue outline indicating that it has focus.
3) When a file upload box or trash can has focus screen reading software now speaks a special message prompting screen reader users how to interact with these items.  I downloaded the ChromeVox extension for chrome and tested these audibly.

Most of my testing was done in Development->Choice of language gradable.

**Comment by bmcutler** [comment]
@pbailie can you please re-review?  Thanks!

**Pull request merged**